src/components/primitive/drop-down/drop-down.component.styles.ts

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { DropDwonProps } from './drop-down.component.props';
+
+export const DropdownWrapper = styled.div<DropDwonProps>`
+  display: flex;
+  width: ${({ wide }) => (wide ? '40rem' : '100%')} !important;
+  flex-flow: column;
+  justify-content: flex-start;
+`;
+
+export const StyledSelect = styled.select`
```

Comment:
I think we need a border for the select style, color defaulted to primary as the background is plain on all screens. Refer the design.

Reply:
yeah, added in the latest commit.